sdk/storage/azure-storage-queues/inc/azure/storage/queues/queue_sas_builder.hpp

```diff
@@ -48,12 +48,14 @@ namespace Azure { namespace Storage { namespace Sas {
     All = ~0,
   };
 
+  /** @brief Logical OR of two values*/
```

Comment:
This is actually bitwise OR (`|` in C++), not logical OR (`||` in C++)

Reply:
Please propose a suggestion and I'll apply it.